x-pack/test/apm_api_integration/basic/tests/services/top_services.ts

```diff
@@ -99,21 +99,24 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             Array [
               Object {
                 "avgResponseTime": Object {
-                  "value": 556200.153101878,
+                  "value": 420419.34550767,
                 },
                 "transactionErrorRate": Object {
                   "value": 0,
                 },
                 "transactionsPerMinute": Object {
-                  "value": 117.133333333333,
+                  "value": 0.0219138056975895,
                 },
               },
               Object {
                 "avgResponseTime": Object {
-                  "value": 2629229.16666667,
+                  "value": 2382833.33333333,
+                },
+                "transactionErrorRate": Object {
+                  "value": null,
                 },
                 "transactionsPerMinute": Object {
-                  "value": 3.2,
+                  "value": 5,
```

Comment:
I *think* these should not change.

Reply:
The only thing that might change is that `transactionErrorRate` will exist if there is no data for that service.

Reply:
@dgieselaar I believe it makes sense the `transactionsPerMinute` change its value since it's now calculated based on the top transaction type before it didn't take into consideration the transaction type at all.

https://github.com/cauemarcondes/kibana/blob/apm-inventory-page-transaction-type/x-pack/plugins/apm/server/lib/services/get_services/get_service_transaction_stats.ts#L171

Reply:
More than half of the services on the observability test environment have only one transaction type.

Reply:
@dgieselaar Sorry for the huge snapshot here, it shows the number of transaction types per service. The API test will change for the ones with more than one transaction type. But, I also found out a bug and fixed here 

https://github.com/elastic/kibana/pull/86434/commits/0fbf4d5303f69e854014b34c79a81295c38a21ad#diff-b83c835e7e4abd7324ea52a0f588332a6934edde706f526f6ee5c571e5757b80R178

```json
"buckets":[
            {
               "key":"kibana",
               "doc_count":3514,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"taskManager",
                        "doc_count":2145
                     },
                     {
                        "key":"request",
                        "doc_count":1369
                     }
                  ]
               }
            },
            {
               "key":"opbeans-python",
               "doc_count":513,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"celery",
                        "doc_count":457
                     },
                     {
                        "key":"request",
                        "doc_count":56
                     }
                  ]
               }
            },
            {
               "key":"opbeans-node",
               "doc_count":223,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"Worker",
                        "doc_count":128
                     },
                     {
                        "key":"request",
                        "doc_count":95
                     }
                  ]
               }
            },
            {
               "key":"opbeans-ruby",
               "doc_count":134,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"request",
                        "doc_count":134
                     }
                  ]
               }
            },
            {
               "key":"opbeans-go",
               "doc_count":128,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"request",
                        "doc_count":128
                     }
                  ]
               }
            },
            {
               "key":"kibana-frontend",
               "doc_count":96,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"http-request",
                        "doc_count":58
                     },
                     {
                        "key":"user-interaction",
                        "doc_count":18
                     },
                     {
                        "key":"route-change",
                        "doc_count":14
                     },
                     {
                        "key":"page-load",
                        "doc_count":6
                     }
                  ]
               }
            },
            {
               "key":"opbeans-dotnet",
               "doc_count":87,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"request",
                        "doc_count":87
                     }
                  ]
               }
            },
            {
               "key":"opbeans-java",
               "doc_count":75,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"request",
                        "doc_count":75
                     }
                  ]
               }
            },
            {
               "key":"opbeans-rum",
               "doc_count":16,
               "transactionType":{
                  "doc_count_error_upper_bound":0,
                  "sum_other_doc_count":0,
                  "buckets":[
                     {
                        "key":"page-load",
                        "doc_count":16
                     }
                  ]
               }
            }
         ]
```